src/main/java/com/jcabi/email/Protocol.java

```diff
@@ -42,11 +42,13 @@
  * <pre> Postman postman = new Postman.Default(
  *   new SMTP(
  *     new Token("user", "password").access(
- *       new Protocol.SMTPS("smtp.gmail.com", 587)
+ *       new Protocol.SMTP("bind", "port")
```

Comment:
@mkordas Gmail does not work with SMTP protocol through SMTP wire, so the Javadoc was somewhat misleading. It's usually the first thing anyone tries, so you don't want your first example to not work.